<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaskEditingGui</class>
 <widget class="QDialog" name="MaskEditingGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1905</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Review Masks</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_autoSnake">
       <property name="text">
        <string>Automatically Generate Masks for All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_loadAnchorContours">
       <property name="text">
        <string>Load Anchor Contours</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_saveAnchorContours">
       <property name="text">
        <string>Save Anchor Contours</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_loadAllInitContours">
       <property name="text">
        <string>Load Init Contours for All Sections</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_saveAllInitContours">
       <property name="text">
        <string>Save Initial Contours for All Sections</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_saveCurrFinalMasks">
       <property name="text">
        <string>Save Final Mask for Current Section</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_saveAllFinalMasks">
       <property name="text">
        <string>Save Final Masks for All Sections</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_exportAllMasks">
       <property name="text">
        <string>Export Masks as PNG for All Sections</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Merged Mask</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="gview_merged_mask">
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>500</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="button_update_merged_mask">
         <property name="text">
          <string>Update Merged Mask</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Contrast Enhanced Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="gview_thresholded">
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>500</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Channel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_channel"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Initial Submask Contour</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="init_snake_contour_gview">
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>500</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Submasks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="gview_final_masks_user">
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>500</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Shrink</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider_snakeShrink">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_snakeShrink">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Min Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider_minSize">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_minSize">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_snake">
         <property name="text">
          <string>Do Snake</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Min Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider_minSize2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_minSize2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_minSize2">
         <property name="text">
          <string>Eliminate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
